src/v/redpanda/admin/api-doc/usage.json

```diff
@@ -24,8 +23,34 @@
         }
     ],
     "models": {
-        "usage_response": {
-            "id": "usage_response",
+        "follower_usage_response": {
+            "id": "leader_usage_response",
+            "description": "Kafka usage metrics for this node",
+            "properties": {
+                "begin_timestamp": {
+                    "type": "long",
+                    "description": "Timestamp when bucket had opened"
+                },
+                "end_timestamp": {
+                    "type": "long",
+                    "description": "Timestamp when bucket was closed"
+                },
+                "open": {
+                    "type": "boolean",
+                    "description": "If this result contains results from window that is not yet closed"
+                },
+                "kafka_bytes_sent_count": {
+                    "type": "long",
+                    "description": "Number of bytes sent to client via the kafka api"
+                },
+                "kafka_bytes_received_count": {
+                    "type": "long",
+                    "description": "Number of bytes received by redpanda via the kafka api"
+                }
+            }
+        },
+        "leader_usage_response": {
```

Comment:
I originally thought they will be mutually exclusive, so we'll have `global` for values like storage and `per-broker` for values that are supposed to be summed up by whoever is consuming the API. and then leader returns both of these... the advantage of that is that the consumer of this knows what to do with the values (sum or not sum)